src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmersSparkUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceWindowFunctions;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Unit tests for FindBadGenomicKmersSpark
+ */
+public class FindBadGenomicKmersSparkUnitTest extends BaseTest {
+
+    private static final int KMER_SIZE = SVConstants.KMER_SIZE;
+    private static final String REFERENCE_FILE_NAME = hg19MiniReference;
+
+    @Test(groups = "spark")
+    public void badKmersTest() throws IOException {
+        final File refSequenceFile = File.createTempFile("genomic_sequence", "txt");
+        refSequenceFile.deleteOnExit();
```

Comment:
BaseTest just forwards to IOUtils, which is (sorry if I'm goring anyone's ox) poorly designed, IMHO.  Rather than taking a stab at a mess of possible indices to include, it ought to be sub-typed for particular file types with particular indices.  Since it provides so little added value in terms of reducing repetitive code, I'd rather just stick to the standard Java way.  This also has the advantage of being understandable by any Java programmer, rather than just those immersed in the lore of GATK.